rest/route/distro.go

```diff
@@ -742,182 +579,6 @@ func validateDistro(ctx context.Context, apiDistro *model.APIDistro, resourceID
 	return d, nil
 }
 
-///////////////////////////////////////////////////////////////////////
-//
-// PATCH /rest/v2/distros/{distro}/execute
-
-type distroExecuteHandler struct {
-	opts   model.APIDistroScriptOptions
```

Comment:
This struct can be deleted if the CLI logic is deleted.

Reply:
done